test/system/700-play.bats

```diff
@@ -964,6 +964,63 @@ EOF
     done
 }
 
+# bats test_tags=ci:parallel
+@test "podman kube play healthcheck should wait initialDelaySeconds before executing healthcheck" {
```

Comment:
I think it is sufficient to just check that the healtcheck is NOT executed before the time is elapsed, because the test above is testing that the healtcheck is executed